### PR TITLE
CLDR-16757 Core data for bew

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16757

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by pedberg-icu — APPROVED** [review]
Looks good. I see we already have a name for it in `en`, an entry in likelySubtags, and territoryInfo data in ID for it.

**Comment by srl295** [comment]
> Looks good. I see we already have a name for it in `en`, an entry in likelySubtags, and territoryInfo data in ID for it.

Yes, this one came ready to go!

**Comment by macchiati** [comment]
I believe this is incomplete; it doesn't have the name of the language bew.

**Comment by srl295** [comment]
> I believe this is incomplete; it doesn't have the name of the language bew.

The English name was already in en.xml, the autonym is not needed in https://cldr.unicode.org/index/cldr-spec/core-data-for-new-locales nor in the core data form.  The autonym is needed for Basic.

**Comment by macchiati** [comment]
You're right; we don't require (or want) that anymore in core.

**Comment by srl295** [comment]
> You're right; we don't require (or want) that anymore in core.

Tests ought to verify that all locales are at core. Should we have a Ddl ticket for double checking ?

**Comment by grcspace311** [comment]
> CLDR-16757
> 
> - [ ] This PR completes the ticket.
> 
> ALLOW_MANY_COMMITS=true
> 

>echo [] CLASSPATH=%APP_HOME%\gradle\wrapper\

File: common/main/bew.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<!-- Copyright © 1991-2023 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+SPDX-License-Identifier: Unicode-DFS-2016
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
```
Comment (grcspace311) [inline common/main/bew.xml:6]: > CLASSPATH=%APP_HOME%\gradle\wrapper\ 'add'

**Review by grcspace311 — APPROVED** [review]
Grcspace311

**Pull request merged**